Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -173,6 +174,30 @@ def main(cls, args, repository, **kwargs):
         if not rmt:
             sys.stderr.write("'{}' doesn't have a recognized remote\n".format(repository.root_path))
             return 1
+
+        existing_pr = None
+        if rmt.pull_requests:
+            for pr in rmt.pull_requests.find(opened=None, head=repository.branch):
+                existing_pr = pr
+                if existing_pr.opened:
+                    break
+            if existing_pr and not existing_pr.opened and not args.defaults and (args.defaults is False or Terminal.choose(
+                    "'{}' is already associated with '{}', which is closed.\nWould you like to create a new pull-request?".format(
+                        repository.branch, existing_pr,
+                    ), default='No',
+            ) == 'Yes'):
+                existing_pr = None
+
+        # Remove "merging-blocked" label
+        if existing_pr and existing_pr._metadata and existing_pr._metadata.get('issue'):
+            log.info("Checking PR labels for '{}'...".format(cls.BLOCKED_LABEL))
```

Comment:
Do we need to print this at info level?

Reply:
In this script, the `info` level is the first `-v`, so unless folks are adding logging flags, they won't see this line.